packages/select/src/components/query-list/queryList.tsx

```diff
@@ -417,11 +417,18 @@ export class QueryList<T> extends AbstractComponent2<QueryListProps<T>, IQueryLi
     private getActiveElement() {
         const { activeItem } = this.state;
         if (this.itemsParentRef != null) {
+            const { getActiveElement } = this.props;
```

Comment:
I figured that instead of delegating entirely to `props.getActiveElement()` (the user would then have to re-implement the `isCreateNewItem` checks and need access to `state.filteredItems` for example), it was more ergonomic to delegate to it the way I did.